app/core/speaker_diarization.py

```
@@ -0,0 +1,337 @@
+                        'end_time': result['end'],
+                        'speaker_tag': result['speaker_id']
+                    })
+                
+                return converted_results
+                
+            finally:
+                # 4. 임시 파일 삭제
+                if os.path.exists(audio_path):
+                    os.unlink(audio_path)
+                    
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)